integration/tests/cmd/stream/test_stream_delete_command.rs

```diff
@@ -1,46 +1,37 @@
-use crate::cmd::common::{IggyCmdCommand, IggyCmdTest, IggyCmdTestCase};
+use crate::cmd::common::{IggyCmdCommand, IggyCmdTest, IggyCmdTestCase, TestStreamId};
 use assert_cmd::assert::Assert;
 use async_trait::async_trait;
+use iggy::client::Client;
 use iggy::streams::create_stream::CreateStream;
-use iggy::streams::get_stream::GetStream;
-use iggy::{client::Client, identifier::Identifier};
+use iggy::streams::get_streams::GetStreams;
 use predicates::str::diff;
 use serial_test::serial;
 
-enum TestStreamId {
```

Comment:
This is used in one file but with tests which are under development it will be used in multiple files - I propose to keep it here.

Reply:
I didnt change the code, just renamed files.